subsys/bluetooth/mesh/Kconfig

```diff
@@ -378,19 +378,27 @@ config BT_MESH_IVU_DIVIDER
 config BT_MESH_TX_SEG_MSG_COUNT
 	int "Maximum number of simultaneous outgoing segmented messages"
 	default 1
-	range 1 255
+	range 0 255
 	help
 	  Maximum number of simultaneous outgoing multi-segment and/or
 	  reliable messages.
 
+	  Note that: Since Mesh Segmentation\reassemblying is a mandatory
```

Comment:
minor: please use a forward slash (`/`) instead of a backslash (`\`). Backslash is often used as an escape character, so it makes me a bit uneasy seeing it in such context.